### PR TITLE
ADS release v2.8.8

### PR DESCRIPTION
**Pull request by liudmylaru**
### Description

Running release v2.8.8
Jira: https://jira.oci.oraclecorp.com/browse/ODSC-44376

- added release notes
- updated ADS version
- updated trigger for readthedocs documentation build

### Pre-commit check
```
check python ast.....................................(no files to check)Skipped
check docstring is first.............................(no files to check)Skipped
check json...............................................................Passed
check for merge conflicts................................................Passed
check yaml...............................................................Passed
detect private key.......................................................Passed
fix end of files.........................................................Passed
pretty format json.......................................................Passed
trim trailing whitespace.................................................Passed
black................................................(no files to check)Skipped
rst ``code`` is two backticks............................................Passed
rst ``inline code`` next to normal text..................................Passed
Detect hardcoded secrets.................................................Passed
check-copyright......................................(no files to check)Skipped
```

### CODE REVIEW

**Review by lu-ohai — DISMISSED** [review]

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,17 @@
 Release Notes
 =============
 
+2.8.8
+-----
+Release date: July 27, 2023
+
+* Added ``PyTorchDistributed`` Runtime for Data Science Jobs to support training large language models with PyTorch.
+* Added options to configure flexible shape in OPCTL.
```
Comment (lvallad) [inline docs/source/release_notes.rst:10]:
```suggestion
* Added options to configure flexible shape in ``OPCTL``.
```

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+* Refactored code in ``deploy()`` in ``GenericModel`` to prioritize the parameters.
```
Comment (lvallad) [inline docs/source/release_notes.rst:11]:
```suggestion
* Refactored ``deploy()`` in ``GenericModel`` to prioritize the parameters.
```

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+* Fixed opctl commands delete/cancel/watch/activate/deactivate to add missing parameter options.
```
Comment (lvallad) [inline docs/source/release_notes.rst:12]:
```suggestion
* Fixed the ``opctl`` delete/cancel/watch/activate/deactivate commands to add missing parameter options.
```

File: docs/source/release_notes.rst [hunk 4]
```diff
@@ -2,6 +2,17 @@
...
+* Fixed opctl command to call run to start ml job if no YAML is passed in.
```
Comment (lvallad) [inline docs/source/release_notes.rst:13]:
```suggestion
* Fixed the ``opctl`` command to call run to start an ML job when no YAML is specified.
```

File: docs/source/release_notes.rst [hunk 5]
```diff
@@ -2,6 +2,17 @@
...
+* Deprecated ``DatasetFactory`` class and refactored code.
```
Comment (lvallad) [inline docs/source/release_notes.rst:14]:
```suggestion
* Deprecated the ``DatasetFactory`` class, and refactored the code.
```

**Review by lu-ohai — APPROVED** [review]

**Review by mrDzurb — APPROVED** [review]

File: docs/source/release_notes.rst [hunk 6]
```diff
@@ -2,6 +2,17 @@
 Release Notes
 =============
 
+2.8.8
+-----
+Release date: July 27, 2023
+
+* Added ``PyTorchDistributed`` Runtime for Data Science Jobs to support training large language models with PyTorch.
```
[the remark this answers is not in the document]
Comment (liudmylaru) [reply docs/source/release_notes.rst]: fixed

File: docs/source/release_notes.rst (hunk 2 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: fixed

File: docs/source/release_notes.rst (hunk 4 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: fixed

File: docs/source/release_notes.rst (hunk 5 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: fixed

File: docs/source/release_notes.rst (hunk 3 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: fixed

File: docs/source/release_notes.rst (hunk 1 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: fixed

**Pull request merged**